community/kernel/src/main/java/org/neo4j/kernel/api/exceptions/UnauthorizedAccessKernelException.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2002-2016 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.api.exceptions;
+
+public class UnauthorizedAccessKernelException extends KernelException
+{
+    public UnauthorizedAccessKernelException( String message )
+    {
+        super( Status.Transaction.UnauthorizedAccess, (Throwable) null, message );
```

Comment:
why do we need to puss  `(Throwable) null`? There is another constructor without `Throwable`